docs/admin/install/README.md

```diff
@@ -24,7 +24,7 @@ showlandingtoc: "false"
 # Installing Knative
 
 !!! tip
-    You can install a local distribution of Knative for development use, by following the [Getting started guide](../getting-started/getting-started.md){_blank}.
+    You can install a local distribution of Knative for development use by following the [Getting started guide](../getting-started/){_blank}.
```

Comment:
```suggestion
    You can install a local distribution of Knative for development use by following the [Getting started guide](../getting-started/). 
```
I think I may have been the one to do this, but internal link = same tab, external link = new tab

@RichardJJG can you add this to the style guide if it's not already there?